.travis.yml

```diff
@@ -9,8 +9,6 @@ branches:
 
 php:
   - 5.5
-  - 5.6
```

Comment:
we test 5.5 but not 5.6 now?

Reply:
@cocojoe What's the point on testing each version that comes out? Isn't php retro-compatible? IMO, just test the minimum supported version and the latest supported one. Each build runs for ~8mins. Multiply that for each version and flavor you want to test and you get how much it takes for a php PR to run CI.